testonly/integration/clientserver.go

```diff
@@ -74,14 +83,13 @@ func listen() (string, net.Listener, error) {
 
 // getTestDB drops and recreates the test database.
 // Returns a database connection to the test database.
-func getTestDB(testID string) (*sql.DB, error) {
+func getTestDB(testID string) (*sql.DB, string, error) {
```

Comment:
We were trying to isolate the db connection string away from other functions and just return a proper `sql.DB`. Is there a reason why we can't maintain this abstraction?